### ISSUE TITLE
AceJump  dosen't work in ideavim sometimes

### ISSUE THREAD

**Issue by hujianxin** [johnlindquist/AceJump#107]
I binded AceJumpAction to `<Space>jj` and AceJumpLineAction to `<Space>jl`. When I hit `<Space>jj` too quickly, AceJumpAction will not work, however AceJumpLineAction always works well.

IntelliJ IDEA version is 2016.2.6.

**Comment by hujianxin** [comment]
If I change AceJumpAction from `<Space>jj` to `f` , this problem will disappear. But I really like `<Space>jj`

**Comment by hujianxin** [comment]
My fault, binding it to `f` also causes this problem.

**Comment by thirtythreeforty** [comment]
Yep, I'm seeing this too with it bound to `s`.  It should be noted that `Ctrl-;` still works fine, though.

**Comment by thirtythreeforty** [comment]
This might actually be a duplicate of #77.

**Comment by breandan** [comment]
There appears to be a considerable delay when remapping IdeaVim keys to AceJump. This may be due to the [TypedActionHandler](https://github.com/JetBrains/ideavim/blob/master/src/com/maddyhome/idea/vim/VimTypedActionHandler.java) delegation as Andrey [mentions](https://github.com/johnlindquist/AceJump/issues/77#issuecomment-256316791), but will need further investigation before we can issue a fix. Seems like @hughnguy reported the issue first, but let's leave both tickets open in case they are unrelated. Thanks @hujianxin and @thirtythreeforty for corroborating.

**Comment by hujianxin** [comment]
@breandan Thanks for your reply, there is something wrong in my description. `<Space>jl` for `AceJumpLineAction` is alse not working.

**Comment by hujianxin** [comment]
I have used `emacsIDEA` plugin. This plugin also has a function like `AceJumpAction`. But the problem described above does not appear when using it.

**Comment by breandan** [comment]
As mentioned [here](https://github.com/johnlindquist/AceJump/issues/77#issuecomment-270268856) I recently made a bunch of changes to support IdeaVim compatibility, and I have not witnessed this issue since. 

@hujianxin @thirtythreeforty Can you reproduce this behavior on the latest version of AceJump? Thanks.

**Comment by hujianxin** [comment]
@breandan I just tested this on Intellij 2016.3.2 and latest ideavim, and I couldn't reproduce it anymore, This bug maybe disappear. Thanks for your great work!

In fact, I have not used ideavim for long time.

**Comment by breandan** [comment]
Glad to hear, thanks for your feedback. Feel free to reopen if issue persists.

**Comment by thirtythreeforty** [comment]
Just wanted to chime in and confirm that the issue seems to be fixed for me as well (I use IdeaVim daily).  Thanks!
